teaser/logic/buildingobjects/calculation/four_element.py

```diff
@@ -530,6 +532,7 @@ def __init__(self, thermal_zone, merge_windows, t_bt):
 
         # UA-Value
         self.ua_value_win = 0.0
+        self.u_value_win = 0.0
```

Comment:
although for this value there shouldn't be any difference, but we should also add this value to set_calc_default function of each calculation module